backend/curriculum_tracking/tests/test_activity_log_entry_creators.py

```diff
@@ -155,8 +155,31 @@ def test_start_project(self):
 # class log_card_review_request_cancelled_Tests(TestCase):
 
 
-# class log_card_moved_to_complete_Tests(TestCase):
+class log_card_moved_to_complete_Tests(TestCase):
+    LIST_URL_NAME = "agilecard-list"
+    SUPPRESS_TEST_POST_TO_CREATE = True
+    SUPPRESS_TEST_GET_LIST = True
+
+    def test_card_moved_to_complete_called(
+        self, log_card_moved_to_complete):
+        actor_user = UserFactory(is_superuser=True)
+        card = AgileCardFactory(
+            status=AgileCard.IN_REVIEW,
+            content_item=ProjectContentItemFactory(
+                project_submission_type=ContentItem.LINK, template_repo=None
+            ),
+        )
+        self.login(actor_user)
 
+        add_review_url = f"{self.get_instance_url(card.id)}add_review/"
```

Comment:
Where is `self.get_instance_url` defined?  It's defined on a class that you didn't inherit from. So you will get an error here.

I don't think this test will pass. Are you sure that this test runs and that it passes?